query/stdlib/testing/end_to_end_test.go

```diff
@@ -151,32 +151,47 @@ func makeTestPackage(file *ast.File) *ast.Package {
 	return pkg
 }
 
-var optionsSource = `
+// This options definition puts to() in the path of the CSV input. The tests
+// get run in this case and they would normally pass, if we checked the
+// results, but don't look at them.
+var writeOptSource = `
 import "testing"
 import c "csv"
-import "experimental"
 
-// Options bucket and org are defined dynamically per test
+option testing.loadStorage = (csv) => {
+	return c.from(csv: csv) |> to(bucket: bucket, org: org)
+}
+`
+
+// This options definition is for the second run, the test run. It loads the
+// data from previously written bucket. We check the results after runnig this
+// second pass and report on them.
+var readOptSource = `
+import "testing"
+import c "csv"
 
 option testing.loadStorage = (csv) => {
-	return experimental.chain(
-		first:  c.from(csv: csv) |> to(bucket: bucket, org: org),
-		second: from(bucket:bucket)
-	)
+	return from(bucket: bucket)
 }
 `
-var optionsAST *ast.File
 
-func init() {
+var writeOptAST *ast.File
+var readOptAST *ast.File
+
+func prepareOptions(optionsSource string) *ast.File {
 	pkg := parser.ParseSource(optionsSource)
 	if ast.Check(pkg) > 0 {
 		panic(ast.GetError(pkg))
 	}
-	optionsAST = pkg.Files[0]
+	return pkg.Files[0]
 }
 
-func testFlux(t testing.TB, l *launcher.TestLauncher, file *ast.File) {
+func init() {
+	writeOptAST = prepareOptions(writeOptSource)
```

Comment:
Is it possible to initialize a global variable with a function call in Go? I'm pretty sure that it is, and you can just initialize these in their declarations.